FORMAT.md

```diff
@@ -186,6 +228,40 @@ When there is a list of `<elem>` elements, they are assumed to following each ot
 </robot>
 ```
 
+**Interface types:**
```

Comment:
I thought we also wanted to add an <adapter> e lement with some common types? e.g. X5 to X8 adapter

Reply:
Sorry -- this was after another discussion with @daverolli yesterday.  He found the idea of the adapter confusing since this wasn't something that would visually show up as a physical part.  So for now, since the rigid body allows for this if really necessary, we decided to keep the spec slightly simpler and discourage this use case rather than explicitly condone it.